ouroboros-consensus/src/Ouroboros/Consensus/Ledger/Basics.hs

```diff
@@ -906,10 +906,10 @@ extendDbChangelog dblog newState =
     ext (ApplySeqDiffMK sq) (ApplyDiffMK diff) =
       ApplySeqDiffMK $ extendSeqUtxoDiff sq slot diff
 
-pruneDbChangelog ::
+pruneVolatileDbChangelog ::
```

Comment:
The name might be a bit confusing. Do we mean here the volatile DB changelog or the volatile part of the DB changelog?

Reply:
> the volatile DB changelog or the volatile part of the DB changelog

I _think_ I would interpret both of those as the same thing. What distinction did you have in mind?

Reply:
Well, the changelog also contains an immutable part, doesn't it? Everything that comes before the state anchor.